elastic/logstash
# Fix logstash-plugin install command to install non default plugin

## Release notes


Fix the error of `logstash-plugin install` command to install non default plugin.

## What does this PR do?



This commit removes `Bundler.setup` in install path to avoid Gemfile freeze by bundler

## Why is it important/What is the impact to the user?


After the fix of unlocking ecs_compatibility_support version in plugin update (#13218), `logstash-plugin install` has a problem of installing non default plugin. This PR fixes an issue of running `bin/logstash-plugin install logstash-input-google_cloud_storage`

## Checklist




## Author's Checklist



## How to test this PR locally

should be able to install logstash-input-google_cloud_storage 
$ bin/logstash-plugin install logstash-input-google_cloud_storage

## Related issues


- Fixed: #13404

## Use cases



## Screenshots



## Logs
